addon/-task-group.js

```diff
@@ -17,24 +14,15 @@ export const TaskGroup = EmberObject.extend(TaskStateMixin, {
 
   _numRunningOrNumQueued: or('numRunning', 'numQueued'),
   isRunning: bool('_numRunningOrNumQueued'),
-  isQueued:  false
+  isQueued: false,
 });
 
-export class TaskGroupProperty extends _ComputedProperty {
-  constructor(taskFn) {
-    let tp;
-    super(function(_propertyName) {
-      return TaskGroup.create({
-        fn: taskFn,
-        context: this,
-        _origin: this,
-        _taskGroupPath: tp._taskGroupPath,
-        _scheduler: resolveScheduler(tp, this, TaskGroup),
-        _propertyName,
-      });
-    });
-    tp = this;
-  }
+export let TaskGroupProperty;
+
+if (gte('3.10.0')) {
```

Comment:
Mustn't it be `gte('3.10.0-beta.1')` I remember robert once told me ?

Reply:
We actually strip off everything past the main semver version in `ember-compatibility-helpers` for canary, since canary appends the actual commit sha and completely breaka these functions.

Reply:
Right, but we don’t strip the beta bits (and if we are we shouldn’t 😜). I would do:

```suggestion
if (gte('3.10.0-alpha.1’)) {
```

Reply:
We do, but we should deprecate it upstream. I definitely think some users rely on that behavior, but agree it’s not ideal.